Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Public/QTMConnectLiveLinkSource.h

```diff
@@ -18,7 +18,7 @@ class CRTProtocol;
 class QTMCONNECTLIVELINK_API QTMConnectLiveLinkSettings
 {
 public:
-	QTMConnectLiveLinkSettings() : IpAddress("127.0.0.1"), AutoDiscover(false)
+	QTMConnectLiveLinkSettings() : IpAddress("127.0.0.1"), AutoDiscover(false), Stream3d(true), Stream6d(true), StreamSkeleton(true)
```

Comment:
Default all but `StreamSkeleton` to `false`.

Reply:
Rather stream3d to false